src/pages/home/Home.js

```diff
@@ -1,9 +1,16 @@
 import { Alert, Button } from "antd";
 import React from "react";
+import { Helmet } from "react-helmet";
 
 const Home = () => {
   return (
     <>
+      <Helmet>
+        <title>Home | Deuvox</title>
+        <meta name="title" content="Home | Deuvox" />
+        <meta name="description" content="Deuvox is an ecommerce" />
+        <meta charSet="utf-8" />
```

Comment:
Instead of repeatedly specifying meta charset (or other new metas in the future), we can use inheritance concept with React Helmet by specifying these meta and their default values in the Helmet component on App.js. The child components (Home, Login, etc) can override these values. Example:

In App.js:

```
<Helmet
  defaultTitle="Deuvox"
  titleTemplate="%s | Deuvox"
>
  <meta name="description" content="Deuvox is the best" />
  <meta charSet="utf-8" />
</Helmet>
```

In Home.js:

```
<Helmet>
  <title>Home</title>
  <meta name="description" content="Deuvox is an ecommerce" />
</Helmet>
```

When use visit Home page, the title would be `Home | Deuvox` (as it use the titleTemplate) and the description is `Deuvox is an ecommerce`.